Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html <head>
+
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+<title>GD2 PR1</title>
+
+<link rel="stylesheet" <link rel="stylesheet" href="Styles.css">
```

Comment:
Your link tag is broken: 

```suggestion
<link rel="stylesheet" href="Styles.css">
```